components/brave_wallet/common/eth_sign_typed_data_helper.cc

```diff
@@ -209,34 +204,32 @@ std::optional<std::vector<uint8_t>> EthSignTypedDataHelper::EncodeField(
     if (!value.is_list()) {
       return std::nullopt;
     }
-    auto type_split = base::SplitString(type, "[", base::KEEP_WHITESPACE,
-                                        base::SPLIT_WANT_ALL);
+    auto type_split = base::SplitStringPiece(type, "[", base::KEEP_WHITESPACE,
```

Comment:
```suggestion
    const auto type_split = base::SplitStringPiece(type, "[", base::KEEP_WHITESPACE,
```
nit: `const std::vector<std::string_view>`